dash/frontend/src/app/modules/private/pages/cluster/kube-hunter/kube-hunter.component.scss

```diff
@@ -46,3 +45,9 @@
     }
   }
 }
+
+.mat-mdc-card-header {
```

Comment:
Could probably use
- display-xs-flex
- px-xs-3
- mb-xs-1

(or whatever it's supposed to be)

Reply:
@becky-intelletive It's actually not needed. I removed it.